Classifiers/classifier-EWS_v2.json

```diff
@@ -0,0 +1,297 @@
+				},
+				"Email Message ID": {
+					"simple": "message_id",
+					"complex": null
+				},
+				"Email Received": {
+					"simple": "received_representing.email_address",
+					"complex": null
+				},
+				"Email Reply To": {
+					"simple": "",
+					"complex": {
+						"root": "headers",
+						"filters": [
+							[
+								{
+									"operator": "string.isEqual",
+									"left": {
+										"value": {
+											"simple": "headers.name",
+											"complex": null
+										},
+										"isContext": true
+									},
+									"right": {
+										"value": {
+											"simple": "In-Reply-To",
+											"complex": null
+										},
+										"isContext": false
+									},
+									"ignoreCase": false
+								}
+							]
+						],
+						"accessor": "value",
+						"transformers": []
+					}
+				},
+				"Email Return Path": {
+					"simple": "",
+					"complex": {
+						"root": "headers",
+						"filters": [
+							[
+								{
+									"operator": "string.isEqual",
+									"left": {
+										"value": {
+											"simple": "headers.name",
+											"complex": null
+										},
+										"isContext": true
+									},
+									"right": {
+										"value": {
+											"simple": "Return-Path",
+											"complex": null
+										},
+										"isContext": false
+									},
+									"ignoreCase": false
+								}
+							]
+						],
+						"accessor": "value",
+						"transformers": []
+					}
+				},
+				"Email Sender IP": {
+					"simple": "",
+					"complex": {
+						"root": "headers",
+						"filters": [
+							[
+								{
+									"operator": "string.isEqual",
+									"left": {
+										"value": {
+											"simple": "headers.name",
+											"complex": null
+										},
+										"isContext": true
+									},
+									"right": {
+										"value": {
+											"simple": "X-Originating-IP",
+											"complex": null
+										},
+										"isContext": false
+									},
+									"ignoreCase": false
+								}
+							]
+						],
+						"accessor": "value",
+						"transformers": [
+							{
+								"operator": "string.StripChars",
+								"args": {
+									"chars": {
+										"value": {
+											"simple": "[]",
+											"complex": null
+										},
+										"isContext": false
+									}
+								}
+							}
+						]
+					}
+				},
+				"Email Size": {
+					"simple": "size",
+					"complex": null
+				},
+				"Email Subject": {
+					"simple": "subject",
+					"complex": null
+				},
+				"Email To": {
+					"simple": "to_recipients.email_address",
+					"complex": null
+				},
+				"Email To Count": {
+					"simple": "",
+					"complex": {
+						"root": "to_recipients",
+						"filters": [],
+						"accessor": "",
+						"transformers": [
+							{
+								"operator": "general.count",
+								"args": {}
+							}
+						]
+					}
+				}
+			},
+			"dontMapEventToLabels": false
+		}
+	},
+	"unclassifiedCases": {},
+	"custom": true,
+	"transformer": {
```

Comment:
if we know that all should map to "Phishing" we can give an empty `transformer`  (also try for `keyTypeMap` field)